packages/connectors/connector-qq/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@logto/connector-qq",
+  "version": "0.1.0",
+  "description": "QQ web connector implementation.",
+  "author": "Yen Harvey. <2117555041@qq.com>",
+  "dependencies": {
+    "@logto/connector-kit": "workspace:^4.3.0",
+    "got": "^14.0.0",
+    "@silverhand/essentials": "^2.9.1",
+    "ky": "^1.2.3",
+    "zod": "3.24.3"
+  },
+  "main": "./lib/index.js",
+  "module": "./lib/index.js",
+  "exports": "./lib/index.js",
+  "license": "MPL-2.0",
+  "type": "module",
+  "files": [
+    "lib",
+    "docs",
+    "logo.svg",
+    "logo-dark.svg"
```

Comment:
No `logo-dark.svg` provided. So we can just remove it from the files list for now.